at_secondary/at_secondary_server/lib/src/notification/notification_manager_impl.dart

```diff
@@ -47,6 +47,9 @@ class NotificationManager implements NotificationManagerSpec {
     var notificationKeyStore = AtNotificationKeystore.getInstance();
     var notificationResponse = await notificationKeyStore.get(notificationId);
```

Comment:
Can rename the variable to "notification"

Reply:
done